swift/obj/diskfile.py

```diff
@@ -1322,9 +1322,11 @@ def pickle_async_update(self, device, account, container, obj, data,
             os.path.join(async_dir, ohash[-3:], ohash + '-' +
                          Timestamp(timestamp).internal),
             tmp_dir)
+
         if account == '.expiring_objects':
             self.logger.increment('async_pendings_for_expiring_objects')
         else:
+            self.logger.increment('async_pendings.' + account + '.' + container)
```

Comment:
AFAIK the statsd_exporter can only read dot-separated fields as labels, so we would have labels `account` and `container` here. However, I would rather like to have `project_id` as a label so that we have the option of also giving this out to customers in Maia. So I would propose something like this:
```python
self.logger.increment('async_pendings.' + account.replace("AUTH_", "project.") + '.' + container)
```